### PR TITLE
fix errors, move map

### PR DESCRIPTION
**Pull request by e-a-w**
Search Folder: just changed folder name to Search from SearchMap; did not change any actual code in the search files
Map.jsx/map.css: added styling
LocationSearchPage: moved to Map folder, added styling & description
App.jsx: updated new file path for search & map, added scroll to top
ProfileButtons/MailModal: moved Modal to AllProfiles folder, prevent form from opening if not logged in
AccountPage.jsx: hid pets list if the user is a pet sitter

### CODE REVIEW

File: client/src/Account/AccountPage.jsx [hunk 1]
```diff
@@ -25,19 +25,25 @@ const AccountPage = () => {
...
+            <>
+              <Typography className="header-pets" variant="body1">
+                Your Pets:
+              </Typography>
+              <Typography
+                component="div"
+                variant="button"
+                className="header-pets-list"
+              >
+                {currentUser?.owner &&
```
Comment (Stommy123) [inline client/src/Account/AccountPage.jsx:38]: I don't think `currentUser?.owner &&` is needed. You're already checking that on line 28

File: client/src/Account/AccountPage.jsx [hunk 2]
```diff
@@ -25,19 +25,25 @@ const AccountPage = () => {
...
+                  currentUser?.ownedPets?.map((pet) => {
+                    return (
+                      <Link to={`/petprofile/${pet?._id}`}>{pet?.name}</Link>
```
Comment (Stommy123) [inline client/src/Account/AccountPage.jsx:41]: Small and non blocking nit. But `_id` and `name` are required fields aren't they? If thats the case, i don't think need to optionally chain those properties, we already know they'll exist!

File: client/src/Map/Map.jsx [hunk 3]
```diff
@@ -1,151 +1,182 @@
...
-				</Popup>
-			) : null}
-		</ReactMapGL>
-	);
+      {selectedOwner ? (
+        <Popup
+          latitude={selectedOwner.latitude}
+          longitude={selectedOwner.longitude}
+          closeButton={true}
+          closeOnClick={true}
```
Comment (Stommy123) [inline client/src/Map/Map.jsx:152]: Nit: If we want to pass a `true` prop to a component, we actually can do the shorthand. No need to explictly pass `{true}`
```
<Component closeButton closeOnClick />
```

**Review by Stommy123 — APPROVED** [review]
Awesome! Just some small non blocking comments

**Comment by e-a-w** [comment]
> Awesome! Just some small non blocking comments

Thank you! Will implemented the changes now.

**Pull request merged**